### PR TITLE
[SKIP-PREVIEW] ROC-3970 Add CMC users

### PR DESCRIPTION
**Pull request by timja-kainos**

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -67,12 +71,41 @@ module "feature-toggle-api" {
...
+    CMC_ADMIN_PASSWORD          = "${data.azurerm_key_vault_secret.cmc_admin_password.value}"
+    CMC_EDITOR_USERNAME         = "${data.azurerm_key_vault_secret.cmc_editor_username.value}"
+    CMC_EDITOR_PASSWORD         = "${data.azurerm_key_vault_secret.cmc_editor_password.value}"
     // silence the "bad implementation" logs
     LOGBACK_REQUIRE_ALERT_LEVEL = false
     LOGBACK_REQUIRE_ERROR_CODE  = false
   }
 }
 
+data "azurerm_key_vault" "cmc_key_vault" {
```
Comment (nitinprabhu) [inline infrastructure/main.tf]: Is it worth moving cmc related vault stuff like `cmc_key_vault` ,`cmc_admin_username` etc to a different tf file for e.g `cmcUsers.tf`?

File: infrastructure/main.tf [hunk 2]
```diff
@@ -22,6 +22,10 @@ locals {
   nonPreviewVaultName    = "${var.product}-ft-api-${var.env}"
   vaultName              = "${(var.env == "preview" || var.env == "spreview") ? local.previewVaultName : local.nonPreviewVaultName}"
 
+  cmcPreviewVaultName = "cmc-aat"
+  cmcNonPreviewVaultName = "cmc-${var.env}"
+  cmcVaultName = "${(var.env == "preview" || var.env == "spreview") ? local.previewVaultName : local.nonPreviewVaultName}"
```
Comment (doncem) [inline infrastructure/main.tf]: same as `vaultName` - we can re-use that

Comment (timja-kainos) [reply infrastructure/main.tf]: oops, was a typo

File: infrastructure/main.tf (hunk 1 above)
Comment (timja-kainos) [reply infrastructure/main.tf]: done

**Pull request merged**